File: app/src/main/java/com/example/owner_pc/animechecker/contract/AnimeSmallListFragmentContract.java
package com.example.owner_pc.animechecker.contract;

import com.example.owner_pc.animechecker.model.entity.Anime;
import com.example.owner_pc.animechecker.model.entity.AnimePage;

/**
 * Created by owner-PC on 2017/05/29.
 */

public interface AnimeSmallListFragmentContract {

//    void showAnimes(List<AnimeCard> animes);

    void showError();

    void itemClicked(Anime item);
}

File: app/src/main/java/com/example/owner_pc/animechecker/contract/AnimeSmallListFragmentContract.kt
package com.example.owner_pc.animechecker.contract

import com.example.owner_pc.animechecker.model.entity.Anime

/**
 * Created by owner-PC on 2017/05/29.
 */

interface AnimeSmallListFragmentContract {

    //    void showAnimes(List<AnimeCard> animes);

    fun showError()

    fun itemClicked(item: Anime)
}
